axelor-bank-payment/src/main/java/com/axelor/apps/bankpayment/service/bankstatement/BankStatementCreateService.java

```diff
@@ -52,37 +52,30 @@ public BankStatement createBankStatement(
     return bankStatement;
   }
 
-  protected String computeName(BankStatement bankStatement) {
-
-    String name = "";
+  public String computeName(BankStatement bankStatement) {
```

Comment:
StringBuilder is good, but I think it is more appropriate to use StringJoiner to avoid if statement such as `   if (!builder.toString().isEmpty()) {
        builder.append("-");
      }`